getting-started.md

```diff
@@ -1,9 +1,9 @@
 Goerli Nethermind (Execution Client)
-Goerli is ready for the merge!
+Goerli is now ready for the merge!
 
-There are now two APIs in the Geth Execution Client:
+There are now two APIs in Execution Clients:
 
-1. RPC API `http://nethermind-goerli.public.dappnode:8551`. Use this one to query transactions on your node and connect to your web3 wallet.
-2. Engine API `http://nethermind-goerli.public.dappnode:8551`. Use this one to connect your Beacon Chain (Consensus Layer) client.
+1. RPC API `http://nethermind-goerli.public.dappnode:8545`. Use this endpoint to query transactions on your node and connect to your web3 wallet.
```

Comment:
Was wondering if this should be called RPC API because the API engine is also an RPC API.

Reply:
```suggestion
1. Querying API `http://nethermind-goerli.public.dappnode:8545`. Use this endpoint to query transactions on your node and connect to your web3 wallet.
```